upgrading/change-log.md

```diff
@@ -7,6 +7,39 @@ heading: Version 3.15.x
 
 <p class="release-date">2020</p>
 
```

Comment:
New Feature section is missing - It should be present and FC 12 - Events support in chord diagram should be included there.

Reply:
Added